apps/web/src/pages/PoolCandidates/ViewPoolCandidatePage/ViewPoolCandidatePage.tsx

```diff
@@ -65,6 +66,8 @@ const PoolCandidate_SnapshotQuery = graphql(/* GraphQL */ `
   query PoolCandidateSnapshot($poolCandidateId: UUID!) {
     poolCandidate(id: $poolCandidateId) {
       ...CandidateExpiryDateDialog
+      ...RevertFinalDecisionDialog
+      finalDecisionAt
```

Comment:
I think it is safe to remove this since it is not used in this file and already queried in your fragment.

Reply:
Forgot that thanks! Fixed by [4071136](https://github.com/GCTC-NTGC/gc-digital-talent/pull/10152/commits/407113659c1ffe8adf0cbe8fa7c8f8e97ff122c2)